fix: map attribution and other styling

bwees opened the pull request:
Related to #20459 but sets the attribution position on new timeline

bwees committed:
chore: map widget and page styling

shenlong-tanwen reviewed (commented):
Can we also update the background of the bottom sheet drag handle to match the background of the timeline? It feels weird to have two different shades of grey there

bwees committed:
fix: map bottom sheet styling

bwees committed:
fix: attribution location on android

it appears that on android, the attribution marker is positioned from the top of the display and on iOS its positioned from the safe area edge
